.gitlab/kitchen_testing/ubuntu.yml

```diff
@@ -157,3 +157,11 @@ kitchen_ubuntu_upgrade7_iot_agent-a7:
   extends:
     - .kitchen_scenario_ubuntu_a7_x64
     - .kitchen_test_upgrade7_iot_agent
+
+kitchen_ubuntu_process_agent-a7:
+  variables:
+    KITCHEN_OSVERS: "ubuntu-20-04"
+    DEFAULT_KITCHEN_OSVERS: "ubuntu-20-04"
```

Comment:
Just as a curiosity, why do we need to override these env vars and not rely on the values set in the `.kitchen_scenario_*`? Aren't we limiting our tests to a specific OS version?

Reply:
I put these as sample values. If you need to test on every OS version we support, you can remove these, at the cost of a heavier load on our test infrastructure (which is already kind of crowded at the moment).

Reply:
Thanks for clarifying it, Kylian! I think that it's ok to sample the tests in this case. We're already extending our test coverage with these integration tests so this should not be a problem.